pkg/cmd/kafka/update/update.go

```diff
@@ -49,6 +49,10 @@ type options struct {
 	Context    context.Context
 }
 
+var (
+	validAuthOptions = []string{"true", "false"}
```

Comment:
I would prefer if we declare tribool type in utils with those values rater than define something like this in the command